src/tests/ftest/cart/iv_client.c

```diff
@@ -67,6 +69,9 @@ print_usage(const char *err_msg)
 		"\t-x <value>     : Value as hex string, only used for update operation\n"
 		"\t-s <strategy>  : One of ['none', 'eager_update', 'lazy_update', 'eager_notify', 'lazy_notify']\n"
 		"\t-l <log.txt>   : Print results to log file instead of stdout\n"
+		"\t-m <value>     : Value as string, used to control timig to chage group verion\n"
```

Comment:
A few errors here but it's 'chage' which is confusing the system.
```suggestion
		"\t-m <value>     : Value as string, used to control timing to change group version\n"
```

Reply:
Fix with next push.